ParkingFinder/handlers/user_information.py

```diff
@@ -41,6 +50,92 @@ def get(self, user_id):
                 'error': "BAD REQUEST Invalid User Id"
             })
 
+    @coroutine
+    def post(self, user_id):
+        """
+        handle the post request from /user/{user_id}?access_token={access_token}
+        and use the information in the request body to update the database properly
+        simple code to send post request and body json format
+        curl -i -d
+        '{
+            "activated_vehicle":"laosiji",
+            "new_vehicle" :{
+                "plate":"laosiji",
+                "brand":"toyoto",
+                "model":"hongqi",
+                "color":"blue",
+                "year":"1950"
+                }
+        }'
+        http://localhost:8888/user/{user_id}?access_token={access_token}
+
+        :param String user_id:
+        :raise: NotFound:
+        :raise: AssertionError
+        :return:
+        """
+        # set to default to avoid missingArgumentException
+        access_token = self.get_argument(name='access_token', default=None)
+        # TODO: token validation
+        try:
+            assert user_id
+            user = yield UserService.get_user_detail(
+                user_id=user_id
+            )
+            
+            # json.loads() for string or unicode
+            # json load() for file
+            request_body = json.loads(self.request.body)
+            activated_vehicle_plate = request_body.get('activated_vehicle')
+            new_vehicle = request_body.get('new_vehicle', False)
+            flow_check = True
+            if new_vehicle:
+                if new_vehicle['plate'] and new_vehicle['brand'] and new_vehicle['model'] \
```

Comment:
you can build the vehicle entity first, and use vehicle.validate() to validate all required params